desktop/src/gui/menu_bar.rs

```diff
@@ -135,6 +136,17 @@ impl MenuBar {
                                 player.debug_ui().queue_message(DebugMessage::TrackStage);
                             }
                         }
+                        if let Some(player) = &mut player {
```

Comment:
I made this a button that's disabled if the root movie clip doesn't exist.
That makes the code for it stick out compared to the other buttons, but hopefully that's not an issue.

I can make it a "regular" button if desired, but then the user might click the button expecting something to happen,
but nothing happens if the root movie clip doesn't exist.
There could be some kind of popup message indicating that the root clip doesn't exist though.
But a disabled button is probably the cleanest solution from a UX standpoint.

Reply:
> That makes the code for it stick out compared to the other buttons, but hopefully that's not an issue.

It's not only not an issue but also the preferred way to do it!

You can use `on_hover_text_at_pointer` with an explanation when the button is disabled, but I think it's fine.